salesforce/bulk.go

```diff
@@ -52,10 +53,15 @@ type BulkWriteParams struct {
 	Mode BulkWriteMode
 }
 
+type BulkWriteParams BulkOperationParams
```

Comment:
@laurenzlong  I think using BulkWriteXX to all different operation is going to be confusing, ex) bulk query might also use the same parameter struct. I think we can call it `BulkOperationXX` instead. what do you think?

Reply:
operations: delete, upsert, query, ...

Reply:
name sounds good, no need to alias it as discussed.